### PR TITLE
Limit Mean and Median to show 2DP

### PR DESCRIPTION
**Pull request by delishad21**
image
Currently, it is possible for the mean and median feature of the application to display numbers like these. Lets change it so that it is limited to 2 decimal places.

### CODE REVIEW

**Review by Zer0Legion — COMMENTED** [review]
Good fix with the use of built-in libraries!

File: src/main/java/seedu/address/model/ScoreStatistics.java [hunk 1]
```diff
@@ -36,7 +38,8 @@ public String toString() {
         if (mean == -1) {
             return "No scores available";
         }
-        return "Mean: " + mean
-                + ", Median: " + median;
+        DecimalFormat df = new DecimalFormat("0.##");
```
Comment (jayllo-c) [inline src/main/java/seedu/address/model/ScoreStatistics.java:41]: Nice use of the DecimalFormat class

**Review by jayllo-c — COMMENTED** [review]
Good job implementing the function, which would clean up the UI!

**Review by Pughal77 — COMMENTED** [review]
Noce! This would reduce the number of bugs in the app!

**Pull request merged**